jax/core.py

```diff
@@ -54,6 +54,8 @@ def skipping_checks():
 zip = safe_zip
 map = safe_map
 
+pp_jaxpr_eqn_lhs_wrap_threshold_length = 10
```

Comment:
IMO this can be inlined; it's only consumed in one place and it's pretty easy to find.

Reply:
Should it be more like 3 or 5 instead of 10?